src/function.ts

```diff
@@ -140,6 +275,78 @@ abstract class FunctionBase<P, O>
       ResultSelector: "$.Payload",
     };
   }
+
+  protected static normalizeAsyncDestination<P, O>(
+    destination:
+      | FunctionAsyncOnSuccessDestination<P, O>
+      | FunctionAsyncOnFailureDestination<P>
+  ) {
+    return destination === undefined
+      ? undefined
+      : isEventBus<any>(destination)
+      ? new EventBridgeDestination(destination.bus)
+      : isFunction(destination)
+      ? new LambdaDestination(destination.resource)
+      : destination;
+  }
+
+  configureAsyncInvoke(config: EventInvokeConfigOptions<P, O>): void {
```

Comment:
Should this be public or private? I didn't see it in the interface so private?

Reply:
Its in the interface, made it public

![image](https://user-images.githubusercontent.com/289213/171630334-4f3fa650-cd7b-4c57-abc6-374f613c2be1.png)

Reply:
Added an es-lint rule to error on these cases to make your life easier.